Makefile.lb

```diff
@@ -10,7 +10,7 @@ build:
 	cd $(NDCTL_DIR) && ./autogen.sh
 	cd $(NDCTL_DIR) && ./configure CFLAGS='-g -O0' --prefix=$(NDCTL_UTIL_INSTALL_DIR) --sysconfdir=/etc --libdir=$(NDCTL_UTIL_INSTALL_DIR)/lib
 	$(Q)$(MAKE)
-	$(Q)$(MAKE) check
+	#$(Q)$(MAKE) check
```

Comment:
Remove this line if we do not intend to use it. 
We can always know what was there before.